### DISCUSSION TITLE
Is there any way to create a report of dependency updates?

### DISCUSSION THREAD

**Discussion by jcook-uptycs** [renovatebot/renovate#22611] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm looking for a way to set renovate to create a report of the dependencies needing updates?

What I'm looking for is the contents the Dependency Dashboard, but in a JSON report file. A SARIF file would be awesome. 

I'm looking to be able to use the report to populate vulnerability dashboards. Right now Renovate is identifying security vulnerabilities. I would also like to be able to include all dependency updates. 

An example of where it would/could be used https://docs.sonarqube.org/9.8/analyzing-source-code/importing-external-issues/importing-issues-from-sarif-reports/

### What you tried so far.

I have looked to see if a debug log would contain all the data.

I have found the log entry with "msg": "packageFiles with updates", but it doesn't contain the security info with it. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by jannylund** [comment]
I would very much like something similar. Since I am using Bitbucket, I don't even have the normal dependency dashboard.

I wonder if one could simply make a feature that reuses the logic of the dependency dashboard and generate a HTML report for each repo. Or maybe even a summary of all repositories that Renovatebot was looking at.

**Reply by reitzig** [reply]
Related: #23961

Can't find anything about security in our logs, but then I'm not sure we have the necessary integrations set up.
